Upload Toggle filter design and description

deernano opened the pull request:
## Description
* Toggle filter addresses the problem of a user frequently accessing filters, repeatedly, and having to select them over and over again. It will allow a user to quickly toggle, an individual filter criteria on and off, and immediately see the results, without having to relocate the filter criteria, via the drop down. It also addresses users knowing or discovering the existing filters and learning what data will be returned when selecting a filter.

## Changes

Upload design overview and design description.
* Change 01 overview.md
* Change 02 design.md
* Change 03 Img

## Contribution Submission Checklist

- [X] A GitHub issue was created and reviewed with the PatternFly community.
- [X] The pattern design overview.md and design.md templates were used to provide all information, relevant to this design.
- [X] Any new files have been added to the folder for that pattern.
- [X] The folder, for brand new patterns, has been titled with the name of the pattern.
- [X] All images are in an "img" folder and embedded images have the correct filepath.
- [X] The overview.md and design.md, for this design have been commited and pushed to the patternfly-design repository, for review.

deernano commented:
@LHinson @Mgranfie @mcarrano  Please help to review the toggle filter. Thank you !

mcarrano commented:
@bleathem Can you please take a look at this PR when you get a chance and let us know what the source of this Travis error is?  Thnx.

cshinn commented:
@mcarrano removing the site.md file from this PR should solve that issue

LHinson commented on pattern-library/forms-and-controls/Toggle filter/site.md:
```diff
@@ -0,0 +1,8 @@
+layout: page-pattern
+overview: pattern-library/forms-and-controls/filter/design/overview.md
+design: pattern-library/forms-and-controls/filter/design/design.md
+code: false
```
I don't think this is actually implemented yet as I don't see the toggle feature in angular.  Nor does our site support ng yet. 

Line 5 should read "code: false" and remove line 6 and 7

LHinson commented:
Thanks for making the catch @cshinn 

@deernano @mcarrano instead of deleting the site.md, just make the inline changes that I've specified.

deernano commented:
@mcarrano @LHinson @Mgranfie 
I removed the references in the document to "The toggle can be used with the current filter pattern."
and updated the text to communicate that the Toggle filter is used alone.
Please review it, thank you!

deernano commented:
@mcarrano @LHinson @Mgranfie  Updated some design decriptions. Refer to the google doc
https://docs.google.com/document/d/1i-7VPI2Ycw1UH4UixJymdxsDxxCuFXeI524s_9UK_iM/edit#
Please review it, thank you.
